sdk/identity/Azure.Identity/tests/EnvironmentCredentialProviderTests.cs

```diff
@@ -73,6 +73,30 @@ public void CredentialConstructionClientCertificate()
 
                 Assert.NotNull(certProvider);
                 Assert.AreEqual("mockcertificatepath", certProvider.CertificatePath);
+                Assert.False(cred.Client._includeX5CClaimHeader);
+            }
+        }
+
+        [NonParallelizable]
+        [Test]
+        public void CredentialConstructionClientCertificateWithX5c()
```

Comment:
Can we add a test to validate the negative case as well?

Reply:
Sure. The other cases are already technically negative cases (see the one above this where I added the validation), but I added one where I explicitly set the value of env variable to "false".

Reply:
They do send false, but I didn't see that they explicitly validate that the SNI cert option isn't set.